cmake/CPackConfig.cmake

```diff
@@ -111,5 +114,23 @@ if(UNIX AND NOT APPLE)
 
 endif()
 
+# Apple macOS
+if(APPLE)
+
+  set(CPACK_BINARY_BUNDLE "ON")
+  set(CPACK_BINARY_DRAGNDROP "ON")
+  set(CPACK_PACKAGE_DIRECTORY "${CMAKE_BINARY_DIR}")
+  set(CPACK_OUTPUT_FILE_PREFIX ${CPACK_PACKAGE_DIRECTORY})
+  set(CPACK_CPU_ARCH ${CMAKE_SYSTEM_PROCESSOR})
+  set(CPACK_SYSTEM_NAME "${CMAKE_SYSTEM_NAME}-${CPACK_CPU_ARCH}")
+  set(CPACK_PACKAGE_FILE_NAME "${CPACK_PACKAGE_NAME}-${FC21_REV_TAG}-${CPACK_SYSTEM_NAME}")
+  set(CPACK_COMPONENTS_GROUPING ALL_COMPONENTS_IN_ONE)
+  set(CPACK_BUNDLE_NAME "Freeciv21 Installer")
```

Comment:
I don't think it's an *Installer*, it's just Freeciv21.

Reply:
Ok let me get it working and then can fine tune. Using CI as a test bed since I don't have a mac.